### DISCUSSION TITLE
Configure Connections usign Config File or Via CLI

### DISCUSSION THREAD

**Discussion by ck3mp** [dbeaver/dbeaver#18762] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I am trying to create a connection inside DBeaver that is stored. Not ephemeral (as it seems its current behaviour is when using the CLI)

(Kind of related to #5874)

**Describe the solution you'd like**
Either a CLI flag to store the connection or ability to create a connection config file which can be "ingested" when dropped into DBeaver.

E.g.

Using this example (current behaviour is this would open the connection but after closing the connection vanishes when reopened)

`-con driver=driver|database=xxx|openConsole=true|folder=SQLite|user=usr|password=mypwd`

New option:

`-con driver=driver|database=xxx|openConsole=true|folder=SQLite|user=usr|password=mypwd --store`

--store could write the config to the DBeaver config so it remains added to the system.

**Describe alternatives you've considered**
Add all connections needed on startup via Bash script.

**Additional context**
The reasoning for this is that I have a script which outputs about 150 connections and I would love to add them all to DBeaver easily so I can work between all my DBs/Users etc. Currently my script adds them all each startup and takes 10+ mins to open.

**Comment by E1izabeth** [comment, marked as answer]
Please, pay attention to -create and -save options https://github.com/dbeaver/dbeaver/wiki/Command-Line#connection-parameters . I believe it solves your issue.
